mqttwarn/services/desktopnotify.py

```
@@ -30,27 +40,32 @@ def plugin(srv: Service, item: ProcessorItem):
     # Get Message
     message = item.message
-    if message and is_json(message):
+    if message is None:
+        raise ValueError("Message is required")
+    if is_json(message):
        data = json.loads(message)
-    else:
+    else:       
+       try:
+          msg = t.cast(t.Dict, message).get('message')
+       except:
```

Ruff (0.15.20)

[error] Do not use bare `except`

(E722)